tests/unit/test_client.py

```diff
@@ -253,6 +254,18 @@ def json(self):
     def content(self):
         return ''
 
+class MockResponseSimpleJson:
```

Comment:
I was not sure why MockResponse had set the text field in the way that it did, so I created my own mock object that replaces it for the test that needed text to provide a valid json string.